cmd/root.go

```diff
@@ -207,7 +209,7 @@ func sshDefault() (string, error) {
 func init() {
 	// command line flags
 
-	RootCmd.Flags().StringVar(&coordinatorContainer, "coordinator-container", "dremio-master-coordinator", "for use with -k8s flag: sets the container name to use to retrieve logs in the coordinators")
+	RootCmd.Flags().StringVar(&coordinatorContainer, "coordinator-container", "dremio-master-coordinator dremio-coordinator", "for use with -k8s flag: sets the container name to use to retrieve logs in the coordinators")
```

Comment:
we use comma separated elsewhere (labels and hosts), it seems inconsistent to use space separated here

Reply:
done, changed this to a comma